### PR TITLE
Fix a few prototypes/declarations in decoder

### PR DESCRIPTION
**Pull request by rockdaboot**

### CODE REVIEW

File: dec/decode.c [hunk 1]
```diff
@@ -762,7 +762,7 @@ static void DecodeBlockTypeWithContext(BrotliState* s,
   s->context_lookup2 = &kContextLookup[kContextLookupOffsets[context_mode + 1]];
 }
 
-BrotliResult WriteRingBuffer(BrotliOutput output,
+static BrotliResult WriteRingBuffer(BrotliOutput output,
```
Comment (szabadka) [inline dec/decode.c]: Please fix the indentation of the arguments (here and below).

You can add static to  BrotliAllocateRingBuffer as well.

**Comment by rockdaboot** [comment]
@szabadka Indentation fixed and BrotliAllocateRingBuffer() made static

File: dec/decode.c [hunk 2]
```diff
@@ -784,9 +784,9 @@ BrotliResult WriteRingBuffer(BrotliOutput output,
   return BROTLI_RESULT_SUCCESS;
 }
 
-BrotliResult BROTLI_NOINLINE CopyUncompressedBlockToOutput(BrotliOutput output,
-                                                           int pos,
-                                                           BrotliState* s) {
+static BrotliResult BROTLI_NOINLINE CopyUncompressedBlockToOutput(BrotliOutput output,
```
Comment (szabadka) [inline dec/decode.c]: I just noticed that this goes beyond the 80-character line limit that is observed in this project. Could you please break the argument list to the next line? Sorry for the nitpicking.

**Comment by rockdaboot** [comment]
@szabadka Sorry, my fault. 80 columns are now respected.

**Pull request merged**